### PR TITLE
Automated models based on openAPI spec.

### PR DESCRIPTION
**Pull request by evetion**

### CODE REVIEW

File: backend/overstroomik_service/auto_models.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    )
+    municipality: Optional[str] = Field(
+        None, description='geocoded adress (municipality)\n'
+    )
+    zipcode: Optional[constr(regex='\d{4}[A-Z]{2}')] = Field(
+        None, description='geocoded zipcode \n'
+    )
+
+
+class Data(BaseModel):
```
Comment (SiggyF) [inline backend/overstroomik_service/auto_models.py:46]: bit vague....

File: backend/overstroomik_service/main.py [hunk 2]
```diff
@@ -20,16 +20,19 @@ async def read_root():
...
-    # location.resolve()
-    # something with data
-    return Response(location=location)
+@app.get("/by_location", response_model=FloodInfo)
+def by_location(
+    search_field: Optional[str] = None,
+    latitude: Optional[float] = None,
+    longitude: Optional[float] = None,
+) -> FloodInfo:
+    return FloodInfo(
```
Comment (SiggyF) [inline backend/overstroomik_service/main.py:29]: Should it do something in between?

**Review by SiggyF — APPROVED** [review]
Minor comment

File: backend/overstroomik_service/main.py (hunk 2 above)
Comment (evetion) [reply backend/overstroomik_service/main.py:29]: I will add a comment that it should do something in between, but there's nothing there yet.

**Pull request merged**